resources/views/admin/databases/index.blade.php

```diff
@@ -36,7 +36,7 @@
         <li><a href="{{ route('admin.databases.new') }}"><i class="fa fa-plus"></i></a></li>
     </ul>
     <div class="tab-content">
-        <div class="tab-pane active" id="tab_databases">
+        <div class="tab-pane {{ Request::input('tab') == 'tab_dbservers' ? '' : 'active' }}" id="tab_databases">
```

Comment:
Alright, that sounds like a bug with the panel that should be looked at. I'm trying to avoid those PHP blocks in the templates themselves.